### ISSUE TITLE
Don't show "Set destination" when one is set already

### ISSUE THREAD

**Issue by keunes** [organicmaps/organicmaps#1333]
Hello

Don't know if this is related to missing maps, but after having set a destination, and I am in

To reproduce:
* Be in 'unknown territory'
* Select a POI and choose "Route to"
* When asked if you want to download a map, choose 'Later'
* See the error: the notification "Add a destination to plan a route" is displayed in the bottom of the screen.

![Screenshot_20210923-140748.png](https://user-images.githubusercontent.com/11229646/134504851-2fe386e4-9c85-4c60-9c00-e6616aeb58a9.png)

![Screenshot_20210923-141542.png](https://user-images.githubusercontent.com/11229646/134505231-d69ac982-6564-40fb-af28-e3438f07bc5a.png)

**Comment by biodranik** [comment]
Yes, a better UX is needed. How would you fix it? We can not build routes without downloaded maps.
